apps/delivery/views.py

```
@@ -0,0 +1,21 @@
+"""Delivery order API views."""
+from rest_framework import permissions, viewsets
+
+from .models import DeliveryOrder
+from .serializers import DeliveryOrderSerializer
+
+
+class DeliveryOrderViewSet(viewsets.ReadOnlyModelViewSet):
+    """Expose delivery orders to customers and vendors."""
+
+    serializer_class = DeliveryOrderSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)